jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/DefaultCredentialRetrievers.java

```diff
@@ -112,10 +115,16 @@ public DefaultCredentialRetrievers setCredentialHelperSuffix(
     if (knownCredentialRetriever != null) {
       credentialRetrievers.add(knownCredentialRetriever);
     }
-    if (credentialHelperSuffix != null) {
-      credentialRetrievers.add(
-          credentialRetrieverFactory.dockerCredentialHelper(
-              DockerCredentialHelper.CREDENTIAL_HELPER_PREFIX + credentialHelperSuffix));
+    if (credentialHelper != null) {
+      if (credentialHelper.contains(FileSystems.getDefault().getSeparator())
+          && Files.exists(Paths.get(credentialHelper))) {
+        credentialRetrievers.add(
+            credentialRetrieverFactory.dockerCredentialHelper(credentialHelper));
+      } else {
+        credentialRetrievers.add(
```

Comment:
I think we shouldn't use the `credentialHelper` as a suffix if it contains a file separator. `docker-credential-some/path/to/file` doesn't seem like something we should try to use as a credential helper.